tester_app/app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,53 @@
+// Core variables and mixins
```

Comment:
// Refer to this file: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss